.github/scripts/sync_labels.py

```diff
@@ -48,8 +48,8 @@ def get_linked_pr_from_issue_number(repo, number):
 
 def get_linked_issues_based_on_pr_body(repo, number):
     pr = repo.get_pull(number)
-    pattern = fr'Fixes:? (?:#|{repo}#|https://github.com/{repo}/issues/)(\d+)'
-    matches = re.findall(pattern, pr.body)
+    pattern = r'(?:fix(?:|es|ed)|resolve(?:|d|s)|close(?:|s|d))?(?:{repo}|https://github.com/{repo}/issues/)?(\d+)'
```

Comment:
without `f`, i think `repo` won't be interpolated.

Reply:
yes, you are right.. it's back